fades/main.py

```diff
@@ -253,6 +255,12 @@ def go(argv):
         create_venv = True
 
     if create_venv:
+        # Check if the requested packages exists in pypi.
+        if not args.no_precheck_availability:
+            if not helpers.check_pypi_exists(indicated_deps):
```

Comment:
When we do this checking, we are really hitting the network. Probably we should log in INFO before doing that? something like "Checking the availabilty of dependnecies in PyPI"?

Reply:
+1